packages/venia-ui/lib/components/AccountChip/accountChip.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { shape, string } from 'prop-types';
+import { Loader, User as AccountIcon } from 'react-feather';
+
+import { useAccountChip } from '@magento/peregrine/lib/talons/AccountChip/useAccountChip';
+import { mergeClasses } from '@magento/venia-ui/lib/classify';
+
+import Icon from '../Icon';
+import defaultClasses from './accountChip.css';
+
+const AccountChip = props => {
+    const { fallbackText } = props;
+
+    const { isLoadingUserName, welcomeMessage } = useAccountChip({
```

Comment:
Not a big deal, but how about

```es6
const talonProps = useAccountChip({
    fallbackText
});

const { isLoadingUserName, welcomeMessage } = talonProps
```

Reply:
Also I feel, `isUserNameLoading` is easier to read than `isLoadingUserName`. The call is yours.

Reply:
Resolved by https://github.com/magento/pwa-studio/pull/2550/commits/91d4b61822a9d7989c5a9b21f96ee5bd1697ba17.

(I didn't end up changing `isLoadingUserName` 😝 )

Reply:
Haha, no worries 😄